### PR TITLE
fix us101 map

### PR DESCRIPTION
**Pull request by qianyi-sun**
a quick fix for the us101 map on master/develop branch

### CODE REVIEW

**Review by RutvikGupta — APPROVED** [review]

**Review by Gamenot — APPROVED** [review]

File: cli/studio.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def _build_single_scenario(clean: bool, allow_offset_map: bool, scenario: str):
     from smarts.core.scenario import Scenario
 
     traffic_histories = Scenario.discover_traffic_histories(scenario_root_str)
-    shift_to_origin = not allow_offset_map or bool(traffic_histories)
+    shift_to_origin = not allow_offset_map
```
Comment (turix2) [inline cli/studio.py:74]: I would go a step further here.  I suggest forcing it to not shift the map when there's traffic history data:
```python
    # don't shift maps for scenarios with traffic histories since history data must line up with map
    shift_to_origin = not allow_offset_map and not bool(traffic_histories)
```

Then, where the `--allow-offset-map` option is documented just above that, you should amend it to be something like:
```python
@click.option(
    "--allow-offset-maps",
    is_flag=True,
    default=False,
    help="Allows road networks (maps) to be offset from the origin. If not specified, a new network file is created if necessary.  Defaults to False except when there's Traffic History data associated with the scenario.",
)
```

File: scenarios/NGSIM/README.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Once that is done, you should be able to build your scenarios in the normal way,
 ```bash
 scl scenario build-all --clean scenarios/NGSIM
 ```
+or the following command to disable generating `shifted_map-AUTOGEN.net.xml`
```
Comment (turix2) [inline scenarios/NGSIM/README.md:19]: I would reword this to:  "or use the `--allow-offset-map` option to prevent auto-shifting the map to the origin, like:  "

(The main reason for the rewording is because the filename `shifted_map-AUTOGEN.net.xml` is encapsulated, i.e., not to be known or cared about by users.)
